zathura/zathura.c

```diff
@@ -1100,21 +1057,12 @@ bool document_open(zathura_t* zathura, const char* path, const char* uri, const
     goto error_free;
   }
 
-  unsigned int most_freq_width, most_freq_height;
-  document_open_page_most_frequent_size(document, &most_freq_width, &most_freq_height);
-
   for (unsigned int page_id = 0; page_id < number_of_pages; page_id++) {
     zathura_page_t* page = zathura_document_get_page(document, page_id);
     if (page == NULL) {
       goto error_free;
     }
 
-    unsigned int cell_height = 0, cell_width = 0;
-    zathura_document_get_cell_size(document, &cell_height, &cell_width);
-    zathura_document_set_cell_size(document, most_freq_height, most_freq_width);
-    zathura_page_set_width(page, most_freq_width);
-    zathura_page_set_height(page, most_freq_height);
```

Comment:
The calls to `zathura_page_set_width/height` can probably go away. But the cell size still needs to be set, IMHO. I need to take a closer look at what we wanted to achieve here, but changing the page size here seems wrong in any case.

Reply:
Hi Sebastian, thanks very much for working on this project.

It seems like the cell size is being initially set here:
https://github.com/pwmt/zathura/blob/c3ffe3e02fc65be6de92dfc25a7f77e42c0b4e0e/zathura/adjustment.c#L16-L24

If I had to guess what the purpose of the code I removed was, is that without it the double page view looks very spaced out because cell_width/height is over the whole document, not page by page. 
So when the cell_width/height is set according to the largest page, that causes the double view to look strange.

Reply:
fyi this issue was also reported at #447 . On my comment I noted the original issue and commit that added those calls to `zathura_page_set_width()` and `zathura_page_set_height()`, in case they provide you with more insight on why they were added! Thank you for working on a fix ::)